homeassistant/components/growatt_server/__init__.py

```diff
@@ -185,9 +194,193 @@ async def async_setup_entry(
     # Set up all the entities
     await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
 
+    # Register services for MIN/TLX devices
+    await _async_register_services(hass, config_entry, device_coordinators)
```

Comment:
We should register this in `async_setup` instead so the availability of the services is not dependent on a working config entry